docker/test/monolith.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+rm -rf build
+bash ./docker/build.sh
+
+TEST_SUITE=unit-test  scripts/travis-test.sh
```

Comment:
Please note that we've switched away from Travis and this script is not there anymore.